features/base/fontIcons/JitsiFontIcons.js

```diff
@@ -0,0 +1,9 @@
+import { createIconSetFromIcoMoon } from 'react-native-vector-icons';
+import icoMoonConfig from './fonts/selection.json';
+
+/**
+ * Creates the Jitsi icon set from the ico moon project config file.
+ */
+const JitsiFontIcons = createIconSetFromIcoMoon(icoMoonConfig);
+
+module.exports = JitsiFontIcons;
```

Comment:
I'll take care of this myself but I'd like to mention it for the future: ES6's import & export are our preferred ways & we fall back to require & module.exports if ES6's import & export don't work.